### ISSUE TITLE
Commands from Platypus are logged in history

### ISSUE THREAD

**Issue by RAF-87** [WangYihang/Platypus#46]
```
# cat /root/.bash_history 
ZNYo3dtcbYNOt9b4
 echo RQ3y2eV3; uname ; echo qpqYIAfb
 echo xcgNrS4G && whoami ; echo 7YLdiFZI
 echo 7beLrQc0 && which python2 ; echo aCH5uwkn
 echo 5b2G5Npm && which python3 ; echo 9o8mTu9N
 echo pPKpsIAF && ls /sys/class/net ; echo BMqqtI4z
 echo b0pgTn4E && ls /sys/class/net/eth0/address ; echo 6tpgrD2j
 echo lFokBgKs && cat /sys/class/net/eth0/address ; echo 1mFQoWqj
 echo gzhoSuVB && ls /sys/class/net/lo/address ; echo uAE7Hlrg
 echo 7FuHpHbV && cat /sys/class/net/lo/address ; echo AjcQHbet
```
Would it be possible to run `unset HISTFILE` before doing all the above?

**Comment by WangYihang** [comment]
Good idea! I will test it soon. Thanks for reporting!

**Comment by WangYihang** [comment]
It works for me.
Please check it out, inform me if any unexpected behavior occurs, thank you!

```
git clone https://github.com/WangYihang/Platypus
cd Platypus
make
cd build
./platypus
```

**Comment by RAF-87** [comment]
It works for me too!

I was thinking, this way we remove the ability of using history with arrow up. Which is a nice thing to have if you don't need to hide yourself.

Perhaps we could make it optional in the config.yml? Something like `history: false` and leave it false as default?

**Comment by RAF-87** [comment]
Seems to be inconsistent between distributions, normally you would get bash commands stored in memory until written to history file. But in this case seems to just disable them completely (which is not bad).
I would leave it personally as you done and just allow it on the config as I said above.
Let me know what you think.

**Comment by RAF-87** [comment]
Something going wrong. Connection gets terminated.. I don't think is related to HISTFILE change.

```
» jump b52b1beba97bfaa7c54e7abeaa3b2f5d
2021/05/18 10:37:40 The current interactive shell is set to: [b52b1beba97bfaa7c54e7abeaa3b2f5d] tcp://xxx.xxx.xxx.xxx:34558 (connected at: 7 minutes ago) [🐧] [true]
(🐧) xxx.xxx.xxx.xxx:34558 [root] » pty
2021/05/18 10:37:43 spawning /bin/bash on the current client
2021/05/18 10:37:43 attcker window size: (378, 63)                                                                                                                                                                                                                                                                                                                                        
2021/05/18 10:37:43 reseting client terminal...                                                                                                                                                                                                                                                                                                                                           
2021/05/18 10:37:43 reseting client SHELL...                                                                                                                                                                                                                                                                                                                                              
2021/05/18 10:37:43 reseting client TERM colors...                                                                                                                                                                                                                                                                                                                                        
2021/05/18 10:37:43 reseting client window size...                                                                                                                                                                                                                                                                                                                                        
(🐧) xxx.xxx.xxx.xxx:34558 [root] » interact
2021/05/18 10:37:45 Interacting with [b52b1beba97bfaa7c54e7abeaa3b2f5d] tcp://xxx.xxx.xxx.xxx:34558 (connected at: 7 minutes ago) [🐧] [true]
2021/05/18 10:37:45 Setting attacker terminal to raw mode                                                                                                                                                                                                                                                                                                                                 
                                                                                                                                                                                                                                                                                                                                                                                          
2021/05/18 10:37:46 Write to client failed, 
»
```

**Comment by WangYihang** [comment]
> Something going wrong. Connection gets terminated.. I don't think is related to HISTFILE change.

I can not reproduce your situation, please provide more detail, eg: 

* Platypus version
* Victim os version
* Victim kernel version
* Victim Linux distribution
* The sequence of operations that you performed on the victim


As I mentioned in issue #38, for stability, I highly recommend you to use Web UI to `Upgrade` the plain session to the encrypted session named `Termite`.

**Comment by WangYihang** [comment]
> Seems to be inconsistent between distributions, normally you would get bash commands stored in memory until written to history file. But in this case seems to just disable them completely (which is not bad).
> I would leave it personally as you done and just allow it on the config as I said above.
> Let me know what you think.

Good idea!

**Comment by WangYihang** [comment]
> Good idea!

Solved in #50.
